spec.html

```diff
@@ -12640,13 +12643,13 @@ <h1>
             1. If ! IsDataDescriptor(_current_) is *true* and ! IsAccessorDescriptor(_Desc_) is *true*, then
               1. If _Desc_ has a [[Configurable]] field, let _configurable_ be _Desc_.[[Configurable]]; else let _configurable_ be _current_.[[Configurable]].
               1. If _Desc_ has a [[Enumerable]] field, let _enumerable_ be _Desc_.[[Enumerable]]; else let _enumerable_ be _current_.[[Enumerable]].
-              1. Replace the property named _P_ of object _O_ with an accessor property having [[Configurable]] and [[Enumerable]] attributes set to _configurable_ and _enumerable_, respectively, and each other attribute set to its corresponding value in _Desc_ if present, otherwise to its <emu-xref href="#table-object-property-attributes">default value</emu-xref>.
+              1. Replace the property named _P_ of object _O_ with an accessor property whose [[Configurable]] and [[Enumerable]] attributes are set to _configurable_ and _enumerable_, respectively, and whose [[Get]] and [[Set]] attributes are set by _Desc_ or default.
```

Comment:
I’m not sure this is clear enough. Why not s/is present/if it has the field?

Reply:
All of those - "or default" isn't clear enough, doesn't have a definition, and thus doesn't clearly suggest its meaning.

Maybe it would be clearer to make a small AO that takes two records and the field name, and performs the logic? This prose has always been a bit unwieldy.

Reply:
> All of those - "or default" [...] doesn't have a definition,

You didn't see the definition in the AO's description?

> Maybe it would be clearer to make a small AO that takes two records and the field name, and performs the logic? This prose has always been a bit unwieldy.

That would involve a alias/parameter whose value is a field name, and using it to:
- check whether a given record (`_Desc_`) has a field of that name, and
- if so, extract its value.

...which are things that the spec currently doesn't do. We could add such a thing, but an early version of PR #2468 tried something similar ("dynamic record field lookup", "record field name indirection"), and it met with some reluctance/opposition.

Reply:
The spec passes around internal slot names as reified values, and has field names in tables as reified values.

Reply:
(but totally fair about the dynamic lookup part)